Gordon360/Services/RecIM/SportService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Models.CCT.Context;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+
+namespace Gordon360.Services.RecIM
+{
+    public class SportService : ISportService
+    {
+        private readonly CCTContext _context;
+
+        public SportService(CCTContext context)
+        {
+            _context = context;
+        }
+
+        public SportViewModel GetSportByID(int sportID)
+        {
+            return _context.Sport
+                    .FirstOrDefault(s => s.ID == sportID);
+        }
+
+        public IEnumerable<SportViewModel> GetSports()
+        {
+            return _context.Sport.Select(s => (SportViewModel)s).AsEnumerable();
+        }
+
+        public async Task<int> PostSport(SportUploadViewModel newSport)
+        {
+            var sport = new Sport
+            {
+                Name = newSport.Name,
+                Description = newSport.Description,
+                Rules = newSport.Rules,
+                Logo = newSport.Logo ?? "defaultSportPicture.jpg" //will be a good idea to make default pictures
```

Comment:
Should you enter the `"defaultSportPicture.jpg"` string into the database rather than `null`? Or should you just return the default whenever a logo field is null in the database? I would argue the second for efficiency and storage use.

Reply:
I agree, I'll make that change. I forgot that we could make default values more than just empty strings or numbers in the DB

Reply:
Hold on, not sure you fully understand my note here. I am suggesting that the database should store Logo as `null` but when the "GET" route is called here, the service method should update the view model to use "defaultSportPicture.jpg" on the fly. Then you're not storing the string a bunch of times in the DB.

Reply:
ahhhh ok I get it.